crates/ruff_linter/src/rules/flake8_pyi/rules/bad_version_info_comparison.rs

```diff
@@ -57,8 +58,59 @@ impl Violation for BadVersionInfoComparison {
     }
 }
 
-/// PYI006
-pub(crate) fn bad_version_info_comparison(checker: &mut Checker, test: &Expr) {
+/// ## What it does
+/// Checks for if-else statements with `sys.version_info` comparisons that use
+/// `<` comparators.
+///
+/// ## Why is this bad?
+/// As a convention, code for newer Python versions should be put first when
+/// writing multiple versions of code while comparing Python versions.
+///
+/// ## Example
+///
+/// ```python
+/// import sys
+///
+/// if sys.version_info < (3, 10):
+///
+///     def read_data(x, *, preserve_order=True):
```

Comment:
I don't think this formatting is the best, but the `mkdocs` check is failing if I change this in any way.

Reply:
I opened https://github.com/astral-sh/ruff/issues/11568 to track this; I've also bumped into this in the past :-)

Reply:
feel free to revert the last commit to format it correctly as needed.